scapy/arch/linux.py

```diff
@@ -547,6 +553,7 @@ def send(self, x):
                     self.outs.sendto(raw(ll(p)), sdto)
             else:
                 raise
```

Comment:
It seems that  `x.sent_time` could not be set if we reach this line.

Reply:
True, but the packet wasn't sent in that cade